tables/tr-g2.tbl

```diff
@@ -0,0 +1,17 @@
+#+locale:tr
+#+type:literary
+#+contraction:full
+#+grade:2
+#+dots:8
+
+# TODO: Please correct the metadata above. It is not meant to be
```

Comment:
I would say keep .ctb for now, because .tbl isn't entirely "official" yet. We're playing with the idea of moving everything to the new extensions but there is some opposition to it.